queries.sql

```diff
@@ -7,4 +7,58 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon','Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5;
 SELECT * FROM animals WHERE neutered='True';
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Transaction 1 */
+
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT species FROM animals;
+ROLLBACK;
+SELECT species FROM animals;
+
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species='pokemon' WHERE name NOT LIKE '%mon';
+SELECT * FROM animals;
```

Comment:
[OPTIONAL]
- [ ] For your code to be clean, and to avoid some redundancy in your code base, the lines of code from line `20` up to `26` can be deleted since it's the exact same thing from line `28` up to `32`.